ui/src/components/shared/Router.tsx

```diff
@@ -26,6 +26,7 @@ const COMPONENTS: Record<RouteID, ComponentType> = {
   credentials: Credentials,
   importSchema: ImportSchema,
   issueCredential: IssueCredential,
+  issuedQR: IssuedQR,
```

Comment:
As we have `credentialLinkQR: CredentialLinkQR` these should be `credentialIssuedQR: CredentialIssuedQR`